content/release-notes/1.23.0.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2020-11-13T13:30:14-08:00
+linktitle: "1.23.0"
+title: "1.23.0"
+kubernetes: "1.17, 1.18, and 1.19"
+---
+
+{{<features>}}
+* Added new template function [KotsVersion](https://kots.io/reference/template-functions/static-context/#kotsversion)
+* Added [collectors](https://troubleshoot.sh/docs/collect/ceph/) and [analyzers](https://troubleshoot.sh/docs/analyze/ceph-status/) for Ceph.
+* Added a [collector](https://troubleshoot.sh/docs/collect/collectd/) for Collectd RRD files.
+{{</features>}}
+
+{{<changes>}}
+* Added a button for redeploying failed deployments.
+* User will be warned if an incompatible version of [Velero](https://kots.io/vendor/snapshots/overview/#velero-version-compatibility) is installed.
+* Improved description of what instance and application snapshots do.
+* Better error handling when [app configuration](https://kots.io/vendor/config/config-screen/) is being modified.
+{{</changes>}}
+
+
+{{<fixes>}}
+* [kURL](https://kots.io/vendor/packaging/using-tls-certs/) based installs will resume TLS cert monitoring after errors.
```

Comment:
Not sure what this means. What's the actual change/fix?

Reply:
kurl proxy monitors the cert used for TLS termination.  if the cert changes, the new one will be used.  previously an error would cause the monitor to terminate.   [this is the commit.](https://github.com/replicatedhq/kots/commit/24b8327014188c3804b938d5c28bf958518b1a94)